templates/app/plugins/support/test/support.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+const fastify = require('fastify')
+const tap = require('tap')
+const { test } = require('fastify-tap')(fastify, tap)
+const support = require('..')
+
+test('support works standalone', async ({ equal, fastify }) => {
+  const plugin = await fastify(support)
```

Comment:
I don't get this `await`. It's unintuitive and it should not be needed.

Reply:
It's short hand for:

```
const plugin = fastify(support)
await fastify.ready()
```

Why do you think it's unintuitive?

Reply:
update - caught up on phone with matteo, this isn't needed, will be updating accordingly